http/sys_wrapping_test.go

```diff
@@ -366,4 +366,19 @@ func TestHTTP_Wrapping(t *testing.T) {
 	}) {
 		t.Fatalf("secret data did not match expected: %#v", secret.Data)
 	}
+
+	// Ensure that wrapping lookup without a client token responds correctly
+	client.ClearToken()
+	secret, err = client.Logical().Read("sys/wrapping/lookup")
+	if secret != nil {
+		t.Fatalf("expected no response: %#v", secret)
+	}
+
+	if err == nil {
+		t.Fatal("expected error")
+	}
+
+	if respError, ok := err.(*api.ResponseError); !ok || respError.StatusCode != 403 {
```

Comment:
Should we use `errors.As` here instead? The go [docs](https://pkg.go.dev/errors#pkg-overview) have some guidance about using `As` over this pattern, "because the former will succeed if err wraps an *fs.PathError." Obviously in this case it would apply to `api.ResponseError` instead.

Reply:
Oh neat! Thanks for sharing. I'll change.